### ISSUE TITLE
release new dart_internal that allows the latest dev sdk (2.5.0-dev.2.0)

### ISSUE THREAD

**Issue by jakemac53** [dart-lang/sdk#37837]
Today when doing a pub get I get the following warning:

```
Overriding the upper bound Dart SDK constraint to <=2.5.0-dev.2.0 for the following packages:

dart_internal

To disable this you can set the PUB_ALLOW_PRERELEASE_SDK system environment variable to `false`, or you can silence this message by setting it to `quiet`.
```

It does let me get a version solve which is good but we should publish a version nonetheless so we  can reduce the noise (this isn't useful for most users).

**Comment by munificent** [comment]
https://dart-review.googlesource.com/c/sdk/+/112922

**Comment by munificent** [comment]
Published dart_internal 0.1.6.
